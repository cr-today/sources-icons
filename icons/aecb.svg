<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="currentColor"><path d="M21.788.09c.644-.14 2.116-.112 2.645.049l.096.03c1.5.48 2.183 1.31 3.044 3.72l.082.234c1.356 3.897 2.61 4.344 6.4 2.209l.47-.267c.76-.428 1.057-.569 1.348-.613l.037-.005c.086-.01.174-.014.274-.014 1.618 0 4.005 2.703 4.733 5.359l.038.138.05.196.024.104.017.078.023.115c.115.61.134 1.326.136 3.51l.002 1.882c.005 1.988.033 2.579.175 3.109l.012.046.014.046q.04.135.091.283l.086.244c.054.158.092.277.092.29 0 .016-.106.03-.236.03h-.235v1.707l.001.277c.01 1.457.088 1.8.454 2.41.009.015-.09.019-.22.008l-.235-.018v2.257c.006 1.767.063 2.297.31 2.664l.02.027q.025.036.053.07l.057.069c.048.06.016.058-.191.028l-.249-.037v9.791l-.144-.032c-.673-.15-.894-.413-.902-1.091v-.547l-.621-.617c-1.035-1.03-1.658-2.195-1.68-3.135l-.001-.123c-.004-.197-.038-.241-.374-.58l-.06-.06c-.704-.708-1.022-1.268-1.146-2.031l-.025-.16c-.046-.284-.085-.339-.332-.606-.458-.494-.896-1.302-1.038-1.91l-.013-.056a1.1 1.1 0 0 0-.174-.406 4.3 4.3 0 0 1-.508-.75l-.208-.374.028.889c.033 1.007.117 1.818.331 3.169l.153.943c.083.534.058.581-.134.534l-.15-.04c-.448-.116-.48-.114-.43.075l.044.174.1.4.038.15c.157.639.333 1.014.723 1.527l.109.14c.274.35.344.378.603.216l.086-.056c.52-.345.766-.245 1.15.483l.079.15c.172.325.223.37.52.54l.159.089c1.34.747 1.258.564 1.254 3.312v.693c.011 2.43.05 2.284-1.327 2.966l-.19.095a29.8 29.8 0 0 1-11.311 3.13l-.327.023c-.8.057-.956.118-1.366.482l-.31.278-.338-.308c-.383-.343-.557-.404-1.246-.46l-.05-.004c-4.347-.34-8.678-1.537-11.982-3.31l-.247-.132c-1-.532-.915-.378-.915-3.283l-.003-1.022.001-.347c.01-1.582.134-1.485 1.193-2.084l.561-.317.262-.519c.356-.707.513-.779 1.143-.442l.116.063c.164.088.297.15.304.14.626-.788.888-1.298 1.127-2.19l.088-.331c.116-.447.071-.411-.241-.394l-.054.003c-.147.006-.26.003-.26-.008 0-.03.132-.472.275-.928l.024-.076.151-.473.184-.558.313.13c.19.084.223.135.268.447l.03.217c.033.237.063.22.361.034l.253-.157.962.323.368-.281c.227-.17.227-.146.148.162l-.012.047c-.09.35-.107.583-.036.506.066-.072.375-1.127.356-1.207l-.002-.004c-.048-.048.608-1.3.88-1.682l.013-.02c1.483-2.028 3.677-2.738 5.602-1.813l.434.209.168-.102c.752-.457 2.194-.607 3.142-.327 1.78.525 3.254 2.32 3.786 4.603l.016.072c.042.189.086.284.117.254.04-.04.017-.235-.071-.603l-.036-.15c-.022-.095.012-.078.294.113l.321.218.475-.159.474-.159.317.218c.17.114.195.113.199.009v-.04c0-.413.094-.613.347-.735l.026-.012c.205-.09.257-.137.234-.21l-.014-.047c-.226-.773-.272-2.054-.275-8.78V20.47l.003-7.795.121.575c.185.876.384 1.33.854 1.93l.148.188c.151.189.175.162.201-.43l.03-.742c.026-.583.065-1.059.09-1.084.08-.08.216.637.299 1.581l.015.172c.089.934.18 1.183.59 1.615l.242.25c.307.313.332.261.333-.835v-.964c.004-.815.022-1.39.045-1.364.157.172.282.865.365 2.024l.005.066c.06.866.177 1.295.477 1.745.187.28.571.635.687.635.022 0 .04-.761.042-1.731l.001-1.06c.005-1.111.035-1.189.21-.847l.048.096.073.151c.086.185.107.298.127.958l.029 1.125c.023.81.049 1.066.145 1.305l.013.03q.031.075.075.158c.142.276.686.757.856.757.023 0 .043-.77.046-1.74l.001-.983c.003-.907.014-1.029.07-.978l.005.005.01.012.018.022c.265.336.32.595.364 1.904l.014.429c.024.663.061 1.225.087 1.303.034.105.05-.654.052-1.72v-.616c-.006-2.047-.053-4.73-.114-4.986-.416-1.719-1.342-3.27-2.61-4.37l-.084-.073c-.831-.711-1.072-.75-1.858-.339l-.353.186c-3.315 1.732-4.545 1.968-6.04 1.22-1.153-.578-1.762-1.47-2.551-3.766l-.054-.158c-.582-1.72-1.057-2.47-1.867-2.943-1.011-.592-3.333-.413-4.173.32l-.118.104.205.348-.277.01c-.579.02-1.134.281-1.414.66l-.023.034c-.205.3-.228.316.228.316h.118l.11.002c1.651.029 2.36.704 2.361 2.24.003 2.398-2.322 3.95-4.995 3.337a11 11 0 0 1-.876-.234l-.08-.026c-.552-.179-1.114-.418-1.903-.81l-.259-.13c-.334-.17-.708-.365-1.136-.592l-.186-.1c-.837-.446-1.083-.409-1.897.285-1.29 1.1-2.211 2.618-2.624 4.324l-.021.09c-.057.246-.105 1.828-.13 3.46l-.008.622c-.02 1.78-.01 3.427.051 3.252l.004-.014c.029-.125.067-.717.089-1.381l.007-.227c.05-1.354.208-2.066.461-2.066.013 0 .02.41.022 1.007v.389l-.005.827v.493c.002.621.012 1.05.028 1.05.087 0 .388-.215.586-.42l.048-.05c.325-.35.386-.604.422-1.848l.018-.66c.022-.717.048-.98.104-1.136l.004-.01c.113-.296.305-.636.36-.636.014 0 .032.754.044 1.59l.01.76.003.583v.578a5 5 0 0 1-.007.254L9.772 18q-.022.137.356-.202l.057-.053c.482-.457.654-.942.731-2.044l.017-.255c.06-.851.119-1.21.26-1.635l.116-.35.016 1.974c.01 1.033.032 1.506.068 1.506.136 0 .738-.661.887-.975l.027-.058.013-.028.012-.028q.015-.034.028-.069l.01-.028c.034-.095.06-.201.088-.359l.016-.101c.02-.136.043-.304.07-.524l.082-.712c.042-.35.102-.702.138-.807l.071-.205.042.155.004.016c.023.126.054.663.07 1.237l.032 1.098.326-.433.072-.097c.433-.596.634-1.085.799-1.942l.026-.142c.03-.162.05 2.428.053 6.755l.006 7.016-.757 1.124c-.46.688-.586.92-.687 1.24l-.006.022c-.25.817-.723 1.67-1.15 2.083l-.033.031c-.15.136-.245.269-.245.34-.006.763-.656 1.969-1.408 2.612l-.055.048c-.156.137-.158.173-.159.496-.001 1.054-.817 2.475-1.966 3.425l-.335.278v.647l-.001.073c-.014.642-.166.796-.849.955l-.144.033v-9.79l-.47.07.18-.3.057-.096.014-.027q.016-.027.028-.053l.019-.04.012-.027.014-.033c.107-.269.136-.613.138-1.56l-.001-.492-.003-.497-.014-1.988-.432-.031.172-.342c.244-.484.284-.796.286-2.386v-1.661h-.29c-.14-.004-.183-.023-.165-.07l.06-.157c.348-.937.39-1.4.395-4.588V14.16c.004-1.807.024-2.455.143-3.004l.017-.075a9 9 0 0 1 .234-.77c.84-2.525 3.062-4.854 4.628-4.854h.497l.785.414c3.768 1.99 5.308 2.34 6.618 1.507 1.217-.776 1.515-2.69.493-3.175-.38-.18-1.453-.314-1.47-.184l-.016.115c-.065.467-.104.44-.3-.082l-.01-.03c-.438-1.17.117-2.24 1.288-2.506l.095-.02c.181-.042.226-.08.32-.254l.013-.025c.29-.546.897-.89 1.983-1.126m6.526 29.205h-.002c-.019.022.433 1.45.958 3.052l.312.947.21.63q.107.322.21.625l.337.998c.61 1.83.593 1.963.274 2.143-.392.221-.994.348-1.5.316l-.188-.012c-.26-.013-.27.006-.296.108-.12.48-1.262.764-1.982.494l-.282-.105-.309.186c-.724.437-1.503.433-2.133-.007l-.039-.028c-.068-.05-.116-.04-.25.056-.59.42-1.524.399-2.144-.05l-.221-.16-.287.106c-.7.262-1.9-.018-1.99-.463l-.03-.15-.426.026c-.728.043-1.643-.244-1.748-.55l-.003-.01-.007-.025-.003-.014c-.046-.236.093-.707.788-2.747l.16-.468.263-.782.345-1.037c.383-1.159.715-2.191.86-2.681l.03-.106c.098-.344.063-.349-.189-.05l-.017.02c-.762.911-2.055 3.22-2.452 4.369l-.042.123c-.104.302-.14.282-.29-.082l-.062-.149c-.131-.309-.169-.274-.443.522l-.138.402c-.43 1.263-.496 1.65-.376 2.36l.041.23c.074.389.128.396.907.71a24 24 0 0 0 6.215 1.554l.424.043c.59.065.711.11.868.276l.138.148.134-.125c.217-.202.324-.236 1.061-.316l.05-.006c2.334-.252 4.19-.711 6.44-1.593l.817-.32.088-.397.02-.096c.138-.682.066-1.145-.414-2.548l-.216-.62c-.17-.48-.205-.516-.346-.298l-.161.26c-.162.255-.171.203-.311-.286-.206-.719-1.312-2.842-1.838-3.528a8 8 0 0 0-.246-.3l-.069-.08-.113-.128c-.19-.213-.362-.389-.387-.387M14.129 38.07l.29-.37-.374-.752c-.132-.26-.164-.252-.283.068l-.178.489c-.25.702-.256.854-.062.898l.057.011c.238.045.268.017.55-.344m19.59-1.329c-.033-.032-.464.84-.464.939 0 .223.54.76.73.727l.05-.01c.23-.05.24-.146-.03-.907l-.143-.397a3 3 0 0 0-.143-.352m1.028-.028h-.46l.032.2c.035.217.433 1.315.504 1.388.06.061.386-.053.494-.172l.015-.016c.065-.083.035-.156-.256-.74zm-21.604 0h-.233l-.468.878-.086.165-.021.043c-.13.263-.11.288-.036.346l.008.006c.148.112.454.191.513.132.046-.046.298-.766.427-1.187l.02-.066a1.4 1.4 0 0 0 .055-.226c.001-.068-.038-.089-.18-.091m11.709-7.834c-.052-.281-.534-.12-.68.227-.066.16-.088 2.148-.076 4.193l.009 1.082c.02 1.82.066 3.431.129 3.515a.976.976 0 0 0 1.412.165l.05-.045.045-.042.013-.013c.146-.15.153-.302.032-1.302l-.152-1.221c-.067-.544-.155-1.28-.245-2.045l-.45-3.82c-.048-.397-.08-.654-.087-.694m-1.592-.029-.06-.035c-.275-.168-.382-.146-.423.093l-.003.023c-.13.91-.27 1.968-.402 3.031l-.13 1.071c-.297 2.485-.523 4.727-.467 4.823.313.529 1.239.574 1.572.075l.018-.03.011-.019c.073-.145.085-.472.11-3.194l.03-3.18.003-.736v-.436c-.008-1.289-.068-1.371-.26-1.486m-1.532-.378-.022-.005c-.064-.018-.117-.022-.117-.01l-.008.039-.152.682-.309 1.361-.093.415-.172.772-.28 1.284c-.469 2.162-.856 4.067-.904 4.486l-.033.315.19.078c.575.241 1.168-.027 1.255-.56l.007-.052.92-7.43c.07-.554.129-1.025.14-1.125l.028-.222H22a1.5 1.5 0 0 1-.273-.027m4.418.53-.075-.326c-.06-.242-.094-.253-.4-.189l-.028.006c-.14.03-.154.047-.142.22l.013.149c.044.467.183 1.699.352 3.137l.217 1.827.159 1.31.134 1.086.048.375c.147 1.168.278 1.372.877 1.371h.134c.62-.01.639-.13.325-1.592L27 32.854c-.098-.456-.215-.986-.332-1.516l-.29-1.31zM11.808 36.36c-.021-.01-.352.658-.463.934-.03.074.479.359.544.305.07-.058.368-.698.413-.886l.005-.027c.017-.09-.015-.12-.197-.198zm24.01-.01c-.024.003-.47.212-.519.244-.017.011.075.235.18.464l.031.066c.098.206.199.4.233.428.072.06.567-.182.535-.262l-.225-.49c-.115-.249-.22-.451-.235-.45m4.25-6.948c-.033-.011-.057 1.732-.066 3.576l-.003.75-.001.513.001.987c.004 1.088.015 1.914.037 1.995l.014.05c.073.217.099-.194.1-2.068l-.002-1.61-.009-1.887c-.011-1.758-.03-2.292-.07-2.306m-32.539.05c-.037.045-.053.5-.055 1.905v1.095l.012 2.857c.01 1.653.03 2.067.07 1.995.04-.07.06-1.488.06-3.099l-.002-1.28c-.01-1.83-.04-3.528-.085-3.473m19.9-.605c-.054-.083-.714-.381-.714-.322l.004.019.064.284.137.578c.097.421.253 1.13.43 1.945l.314 1.47.096.452c.09.422.177.84.258 1.23l.183.878c.37 1.757.438 1.804.92 1.913.16.036.919-.071.989-.14.01-.01-.218-.725-.527-1.661l-.477-1.437a180 180 0 0 1-1.335-4.177l-.085-.284c-.117-.38-.227-.701-.257-.748m-6.743.634.187-.763.01-.05c.03-.149-.03-.143-.39.003l-.03.012-.083.03-.028.011-.025.012c-.117.06-.176.186-.381.838l-.378 1.22a195 195 0 0 1-.694 2.159l-.192.586-.237.717-.296.89a59 59 0 0 0-.53 1.67l-.029.101c-.037.132-.054.207-.048.213.096.088.453.167.752.165l.075-.001c.545-.02.689-.246.925-1.405l.115-.572c.142-.694.327-1.566.514-2.436l.24-1.101c.214-.973.41-1.835.523-2.299m12.305 3.479c-.056-.01-.588.251-.769.377-.02.014.138.494.32.985l.068.18q.032.087.064.168l.032.081c.138.347.301.884.365 1.2l.116.574.355-.551.411.033c.3.025.401.018.375-.025l-.028-.043c-.044-.065-.13-.193-.225-.33l-.036-.053c-.301-.437-.463-.82-.734-1.758l-.045-.155c-.13-.443-.221-.675-.269-.683m-17.916.194-.213-.119c-.278-.152-.26-.082-.463.603l-.064.212c-.292.954-.497 1.412-.87 1.964l-.152.226.725-.071.405.551.073-.436c.037-.225.125-.566.202-.787l.413-1.112.17-.453c.157-.428.103-.402-.226-.578m23.37-4.707c-.019 0-.027.414-.028 1.031v.345c.002.51.008 1.11.017 1.716l.018 1.117c.023 1.127.053 2.06.081 2.147.11.332.339.753.641 1.177l.28.393.02-5.41v-.685c-.004-.996-.022-1.174-.07-1.23l-.004-.005-.005-.005-.023-.016-.283-.186-.2-.129c-.223-.143-.423-.26-.444-.26m-29.254-.1-.007.003a13 13 0 0 0-.53.35l-.509.349v5.765c.006 1.808.054 1.58.357 1.126l.096-.141.036-.054c.545-.824.56-1.113.572-4.46l.002-1.023-.001-.797c-.002-.68-.008-1.119-.016-1.117m27.918-1.1-.305-.288.028 2.646c.016 1.288.045 1.63.22 2.05l.017.039c.044.101.096.209.16.337.113.23.3.533.417.672l.212.255v-5.05l-.222-.187c-.123-.103-.36-.317-.527-.475M10.8 28.859l-.013-1.914-1.02.964.004 5.011.21-.235c.276-.311.599-.972.729-1.492l.037-.162.005-.027c.04-.216.053-.465.054-1.109l-.002-.478zm1.502-2.227.02-.506c.022-.635 0-.675-.293-.386l-.046.046-.138.14-.405.406.027 3.659.218-.445c.429-.874.555-1.454.617-2.914m23.93-.25-.843-.89.029 1.436c.034 1.386.14 1.887.561 2.73l.056.11q.037.073.067.12l.009.014.01.013c.098.126.113-.108.114-1.144v-.569zm3.845-1.67c-.054-.02-.077.695-.073 1.431l.003.285c.01.618.04 1.184.087 1.26l.005.007c.045.057.058-.215.06-1.263v-.728c-.003-.837-.02-.968-.082-.992m-32.508.02h-.006l-.018-.006q-.006 0-.01.002l-.003.002-.002.004c-.013.024-.018.116-.024.412l-.018 1.389-.003.474v.213c.002.41.017.52.049.467l.005-.01c.036-.093.059-.503.07-1.295l.003-.225.004-.542v-.258c-.004-.426-.018-.626-.043-.626zm30.845-.94c-.023 0-.033.37-.034.836v.148l.001.255.005.365c.01.545.029 1.038.055 1.112.058.166.536.605.878.804l.162.095v-2.986l-.275-.138a5 5 0 0 1-.482-.29l-.104-.074a1 1 0 0 0-.206-.127m-29.18 1.109-.016-1.166-.183.137c-.101.075-.342.23-.537.344l-.353.208v2.929l.118-.063c.5-.264.91-.696.955-1.004l.003-.035c.015-.177.02-.756.012-1.35m4.378-1.096-.174.282c-.096.156-.231.32-.301.366l-.127.083.008 2.525.176-.358c.097-.197.237-.613.312-.925l.033-.143c.087-.388.099-.583.088-1.128zm20.39.039c-.074 0 .03 1.617.128 2.021l.05.183c.337 1.23.428 1.186.434-.162l.002-1.35-.123-.08a1.4 1.4 0 0 1-.275-.319l-.064-.098c-.076-.112-.14-.195-.152-.195m2.834-1.203c-.016 0-.024.23-.026.551v.212c.002.697.027 1.639.06 1.774.066.26.27.518.696.882l.293.25v-2.967l-.144-.085a8 8 0 0 1-.47-.33l-.124-.092a3 3 0 0 0-.285-.195m-26.059.003c-.022.002-.15.092-.29.204l-.09.072a8 8 0 0 1-.417.309l-.213.143v1.667c.003.684.02 1.211.04 1.21.094 0 .706-.606.837-.827l.143-.244.017-1.269.002-.196v-.463c-.003-.364-.013-.607-.029-.606m16.471 1.71.012-.03c.077-.182.117-.13.234.265l.112.38.567-.036c.348-.019.306.009-.072.264l-.14.093c-.244.165-.212.177-.068.532l.06.148c.106.28.058.265-.288.002l-.305-.232-.433.336c-.273.21-.282.19-.159-.116l.095-.233c.095-.243.052-.25-.29-.5l-.352-.258.859-.03.056-.221c.03-.121.081-.285.112-.364m-7.83.615.933-.053.132-.492c.096-.347.112-.312.251.228l.074.286.903.03-.744.495.2.458c.157.367.113.34-.226.103l-.348-.243-.382.282c-.301.22-.322.214-.201-.112l.084-.226c.049-.135.083-.24.083-.254 0-.018-.156-.13-.353-.252zm5.912.157.116-.025c.32-.066.39-.055.638.108l.15.101c.362.248.318.268-.385.429-1.46.333-2.75.327-4.063-.02l-.17-.047c-.297-.085-.27-.12.066-.315l.175-.101c.273-.155.309-.154.536-.105.803.172 2.086.161 2.937-.025m10.218-3.63-.1-.092c-.135-.121-.182-.122-.185.222v.045c0 .138.007.325.017.571l.06 1.354.189.385c.104.21.305.508.447.66l.257.275.004-2.841-.274-.224c-.13-.105-.302-.253-.415-.356m-23.167.164v-.081c-.004-.282-.04-.294-.175-.108-.054.074-.26.253-.458.397l-.36.262v2.661l.154-.09c.597-.353.832-1.203.84-3.041m5.01.142-.016-.173c-.028-.293-.007-.291.149-.022l.037.065c.404.706 1.392 1.79 2.049 2.253l.042.03c.462.313.324.62-.28.62-.32.001-1.534-1.197-2.16-2.133l-.09-.134c-.204-.314-.209-.366-.049-.311l.1.036c.213.076.243.044.219-.231m13.126-.748.027-.055c.211-.442.194-.06-.023.67l-.042.14c-.08.277-.04.287.21.122.33-.219.31-.125-.088.433l-.076.104c-1.216 1.664-2.549 2.594-2.656 1.844l-.006-.052c-.014-.165.023-.203.426-.52.822-.647 1.802-1.832 2.228-2.686m-8.901 2.916-.028-4.05 5.363.002c1.734.005 3.008.02 3.008.038l-.002.008c-.015.05-.125.294-.257.569-1.43 2.973-4.708 4.462-7.77 3.529zm-1.142-11.74.001.014c.006.13.01 1.131.013 2.585l.003 2.341v6.21l-.327-.226c-3.62-2.492-3.637-8.205-.073-10.68l.11-.074c.14-.093.263-.17.273-.17m13.669 7.952-.1-.181v1.246c.003.98.043 1.186.303 1.565l.066.094c.228.316.252.244.254-.85V20.78l-.214-.207a2 2 0 0 1-.311-.39m-20.508.418v-.178c-.008-.403-.054-.39-.178-.158a1.5 1.5 0 0 1-.27.335l-.17.149-.019 1.351-.003.33v.096l.001.1c.007.536.061.503.25.254l.047-.062c.26-.36.31-.644.334-1.707l.006-.33zm26.522 1.644.002-.557c0-1.125-.033-1.381-.138-1.108-.013.034-.019.182-.02.39v.16c.006.689.054 1.779.098 1.856l.005.006c.033.04.048-.156.053-.747M7.57 20.497c-.03 0-.048.389-.048 1.094l.002.371c.005.637.024 1.061.046.994l.003-.014c.022-.113.041-.67.044-1.281v-.28c-.002-.57-.018-.884-.047-.884m31.087-.684-.223-.15v1.388c.005 1.147.065 1.2.775 1.56l.272.137v-2.491l-.301-.147a5 5 0 0 1-.523-.297m-29.413.998v-.492c-.005-.623-.03-.674-.196-.566l-.066.045c-.1.07-.33.2-.51.29l-.327.162v1.436c.003.571.017 1.008.032 1.008.097 0 .658-.346.792-.488l.036-.04c.203-.23.237-.431.239-1.355m27.735-2.074-.122-.09.02 1.237c.021 1.07.077 1.214.577 1.595l.379.285.019-1.604c.005-.715-.008-.843-.069-.888l-.132-.084-.285-.187-.194-.13zm-26.192-.134-.206.165a7 7 0 0 1-.51.349l-.304.184v2.454l.144-.075.069-.036c.625-.343.802-.732.824-1.749l.003-.222c.006-.383.005-.737-.003-.842zm20.664.84.015-.08c.036-.17.062-.115.22.334l.144.408.868.03-.468.314c-.257.173-.253.197-.131.518l.062.164c.057.156.1.28.1.292s-.114-.06-.265-.166l-.07-.05c-.335-.238-.348-.278-.539-.137l-.15.114c-.341.258-.357.257-.235-.138l.04-.13c.093-.312.06-.334-.303-.558l-.328-.201.854-.03.078-.264c.04-.134.086-.315.108-.42m-15.421.688.119-.444.015-.054c.092-.324.19-.472.194-.294v.015c0 .03.05.216.11.413l.109.36.85.03-.374.238c-.282.182-.31.23-.21.538l.089.262c.032.097.054.168.054.177 0 .01-.127-.07-.287-.181l-.072-.05c-.152-.102-.284-.175-.305-.166a3 3 0 0 0-.2.127l-.257.172c-.217.142-.24.13-.158-.12l.054-.157c.07-.2.11-.38.088-.4l-.01-.009-.073-.05-.1-.067-.115-.073-.422-.266zm-3.778-2.543-.863.68v2.393l.148-.105.06-.044c.555-.407.648-.665.671-1.715l.004-.235c.003-.153.004-.298.004-.423v-.216a3 3 0 0 0-.006-.176zm24.038.683-.952-.712.02 1.152c.021 1.055.07 1.232.48 1.617l.168.154c.263.239.283.204.284-.645zm-5.843-3.935c-.056-.131.721.251.88.433.407.468.752 3.493.473 4.158l-.025.057c-.027.057-.03.043-.033-.034-.002-.06-.058-.16-.126-.224-.16-.15-.253-.057-.444.465l-.088.237a1 1 0 0 1-.075.18v-.009a7 7 0 0 1 .077-.384c.323-1.48.105-3.14-.639-4.879m-14.264.877.15-.444.652-.335c.284-.143.299-.13.2.07l-.062.122c-.432.857-.633 1.877-.634 3.213v1.084l-.227-.283-.134.287c-.074.159-.135.33-.135.383l-.001.048c-.012.247-.105-.156-.158-.702-.116-1.206.008-2.426.35-3.443m-2.623 1.101-.123.18a1.7 1.7 0 0 1-.303.316l-.18.137.007 1.259c.004.606.026 1.088.05 1.088.13 0 .442-.613.52-1.022.035-.18.054-.66.045-1.135zm20.747.286-.278-.262.028 1.098c.02.753.043.944.131 1.171.124.321.352.687.428.687.029 0 .046-.501.05-1.044l.002-.22-.001-.266c-.004-.444-.017-.82-.04-.85l-.01-.013a8 8 0 0 0-.31-.301M23.19 11.292a6.41 6.41 0 0 1 5.323 1.932c.59.611 1.289 1.683 1.289 1.976 0 .05-.773.064-3.637.065h-4.576v-3.562l.194-.081c.301-.126.952-.279 1.407-.33m-6.023 1.112.049-.217c.071-.305.098-.259.24.227l.13.445h.833l-.374.256c-.281.195-.308.243-.208.548l.08.236c.078.245.042.238-.239.036l-.389-.283-.387.3c-.295.226-.294.195-.135-.218l.157-.407-.802-.546.574-.02c.356-.013.397-.039.47-.357m13.158-.374c0-.165.087-.024.188.303l.128.422.195.003c.107.003.296.02.42.04l.224.035-.683.445.281.731.01.026c.03.072-.05.037-.28-.126l-.2-.145c-.205-.144-.233-.128-.453.044l-.179.138c-.287.22-.29.199-.161-.17l.081-.236a2 2 0 0 0 .063-.206c0-.016-.13-.117-.298-.233l-.396-.268.847-.03.124-.415a3 3 0 0 0 .089-.358m-8.35-2.26.24-.062c.506-.125.545-.096.575.276l.028.373-.575.146a8.3 8.3 0 0 0-3.147 1.547l-.421.335-.658-.015a3 3 0 0 1-.352-.02c0-.007.16-.164.353-.348 1.209-1.146 2.45-1.846 3.957-2.232m2.973-.092c.158-.158 1.549.247 2.524.735l.127.065c1.356.71 2.743 1.929 2.072 1.836l-.021-.003a1.1 1.1 0 0 0-.397.018l-.045.012c-.13.032-.18.012-.424-.18l-.118-.092c-1.012-.805-1.85-1.23-3.041-1.54l-.234-.059c-.37-.097-.434-.134-.461-.232-.058-.208-.05-.493.018-.56m-2.276-.507.94-.03.11-.44c.101-.395.134-.378.25.092l.102.413.538-.02c.118-.005.22-.007.288-.007h.07q.019 0 .02.002a6 6 0 0 1-.341.254l-.16.113c-.19.138-.193.173-.117.393l.06.17c.113.336.096.368-.169.184l-.08-.057q-.308-.22-.371-.173l-.126.092c-.24.171-.5.34-.512.33-.007-.008.034-.137.095-.298l.079-.2c.097-.258.075-.291-.201-.485zm-1.53-7.103-.037-.25 1.013.048c.38.02.384.043.332.209l-.02.066c-.26.87-1.156.82-1.288-.073"/></svg>