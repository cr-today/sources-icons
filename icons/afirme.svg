<svg xmlns="http://www.w3.org/2000/svg" width="48" height="48" viewBox="0 0 48 48" fill="currentColor"><path d="M2.038 38.603c.904 -1.358 20.216 -29.722 20.719 -30.533c.503 -.812 1.333 -1.566 2.28 -.16c.947 1.406 1.38 2.214 1.937 3.08c.667 1.019 -.667 2.132 -1.563 .971c-1.037 -1.324 -1.22 -2.717 -2.364 -.754c-1.06 1.837 -5.25 7.78 -6.18 9.213c-.69 1.08 -.74 3.188 -.264 4.78c.31 .92 .786 1.772 1.404 2.513c.333 .408 .16 .948 -.37 1.65c-.53 .704 -7.695 11.512 -7.695 11.512h-7.214c-1.51 0 -1.486 -1.056 -.69 -2.272zm9.976 2.272c.212 -.082 15.584 -23.28 16.596 -24.844c1.013 -1.565 2.533 -.576 1.637 1.006c-1.189 2.099 -5.231 8.391 -6.021 9.586c-.714 1.089 -.794 2.739 -.528 4.005a6.144 6.144 0 0 0 2.031 3.54l-4.47 6.707h-9.245zm11.355 0c9.938 -14.942 10.018 -15.25 11.047 -16.585c.874 -1.132 2.272 -.163 1.561 1.016c-.71 1.18 -5.367 8.134 -6.028 9.15c-.777 1.203 -.581 4.287 1.548 4.287c2.298 -.027 12.13 -.081 12.13 -.081s-3.382 -4.826 -4.255 -6.337c-.9 -1.617 .82 -2.183 1.557 -1.16c.479 .679 4.318 6.42 5.208 7.578c.688 .942 .37 2.132 -.714 2.132h-22.053z"/></svg>