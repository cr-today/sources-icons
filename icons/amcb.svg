---
version: "0.0.4"
---
<svg>
  <path d="M24 33.643a9.643 9.643 0 1 0 0 -19.286a9.643 9.643 0 0 0 0 19.286zm-22.5 -32.143h32.143v11.812h-32.143v-11.812zm11.812 12.857v32.143h-11.812v-32.143h11.812zm1.045 20.33h32.143v11.813h-32.143v-11.813zm32.143 -33.187v32.143h-11.812v-32.143h11.812z" />
</svg>
