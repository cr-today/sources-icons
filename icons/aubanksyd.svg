---
version: "0.0.4"
---
<svg>
  <path d="M24.469 32.911l12.125 6.952l-12.124 6.637v-13.589zm-13.219 -7.58l12.399 7.166v14.003l-12.399 -6.646v-14.524zm0 -15.58l25.344 14.54v14.551l-25.344 -14.55v-14.542zm12.49 -8.251v14.423l-12.49 -7.213l12.49 -7.21z" />
</svg>
