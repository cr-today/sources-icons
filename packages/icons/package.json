{
  "name": "@fex-to/provider-icons",
  "version": "3.0.4",
  "description": "A set of sources icons.",
  "author": "dejurin",
  "license": "MIT",
  "homepage": "https://fex.to",
  "publishConfig": {
    "access": "public"
  },
  "repository": {
    "type": "git",
    "url": "git+https://github.com/fex-to/provider-icons.git",
    "directory": "packages/icons"
  },
  "main": "dist/cjs/provider-icons.js",
  "main:umd": "dist/umd/provider-icons.js",
  "exports": {
    "./*": [
      "./icons/*"
    ]
  },
  "module": "dist/esm/provider-icons.js",
  "unpkg": "dist/umd/provider-icons.min.js",
  "typings": "dist/provider-icons.d.ts",
  "sideEffects": false,
  "files": [
    "dist/*",
    "provider-nodes.json",
    "icons/*",
    "provider-sprite.svg"
  ],
  "scripts": {
    "build": "npm run clean && npm run copy && npm run build:icons && npm run build:es && npm run build:bundles",
    "build:icons": "node build.mjs",
    "build:es": "babel src -d dist/esm",
    "build:bundles": "rollup -c ./rollup.config.mjs",
    "copy": "npm run copy:license && npm run copy:icons",
    "copy:icons": "mkdir -p ./icons && cp ../../icons/*.svg ./icons/",
    "copy:license": "cp ../../LICENSE ./LICENSE",
    "clean": "rm -rf dist && rm -rf src/icons/*.js && rm -rf icons && rm -rf stats",
    "test": "jest --env=jsdom"
  },
  "devDependencies": {
    "babel-jest": "^29.4.1",
    "jest": "^29.4.1",
    "jest-environment-jsdom": "^29.4.1"
  },
  "jest": {
    "testEnvironmentOptions": {
      "customExportConditions": [
        "node",
        "node-addons"
      ]
    },
    "transform": {
      "^.+\\.js$": "babel-jest"
    }
  }
}