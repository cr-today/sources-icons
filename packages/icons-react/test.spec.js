import { render } from '@testing-library/react'
import { IconFex } from './src/icons.js'
import React from 'react'
import renderer from 'react-test-renderer'

describe('React Icon component', () => {
  test('should render icon component', () => {
    const { container } = render(<IconFex />)
    expect(container.getElementsByTagName('svg').length).toBeGreaterThan(0)
  })

  test('should update svg attributes when there are props passed to the component', () => {
    const { container } = render(<IconFex size={48} color="cyan" />)

    const svg = container.getElementsByTagName('svg')[0]

    expect(svg.getAttribute('width')).toBe('48')
    expect(svg.getAttribute('fill')).toBe('cyan')
  })

  // Jest creates separate file to store snapshots
  test('should match snapshot', () => {
    const icon = renderer.create(<IconFex />).toJSON()
    expect(icon).toMatchSnapshot()
  })
})