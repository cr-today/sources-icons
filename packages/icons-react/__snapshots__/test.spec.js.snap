// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`React Icon component should match snapshot 1`] = `
<svg
  className="sources-icon sources-icon-fex"
  fill="currentColor"
  height={48}
  viewBox="0 0 48 48"
  width={48}
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    d="M40 1.199H7v45h10.358V28.317h11.319l3.33-4.618 2.804-3.908H17.358v-9.592H40z"
  />
</svg>
`;
